spec/features/admin/translations_spec.rb

```diff
@@ -217,21 +217,20 @@
     end
   end
 
-
   context "localization settings" do
     given(:language) { Spree.t(:this_file_language, scope: 'i18n', locale: 'de') }
     given(:french) { Spree.t(:this_file_language, scope: 'i18n', locale: 'fr') }
 
     background do
       create(:store)
-      SolidusI18n::Config.available_locales = [:en, :'pt-BR', :de]
-      visit spree.edit_admin_general_settings_path
+      SolidusI18n::Config.available_locales = [:en, :'pt-BR', :fr]
+      visit spree.admin_locale_path
     end
 
-    scenario "adds german to supported locales" do
-      targetted_select2_search(language, from: '#s2id_supported_locales_')
+    scenario "adds german to available locales" do
+      targetted_select2_search(language, from: '#s2id_available_locales_')
       click_on 'Update'
-      expect(SolidusGlobalize::Config.supported_locales).to include(:de)
+      expect(SolidusI18n::Config.available_locales).to include(:de)
```

Comment:
By changing the spec you actually are testing the feature provided by `solidus_i18n` not the feature added by this very gem. Please use the fix I made in https://github.com/tvdeyen/solidus_globalize/commit/3683e7668d5443a27518b5c519968ccc4f6f2ce9